source/dub/packagesuppliers/fallback.d

```diff
@@ -40,19 +39,33 @@ private template fallback(T, alias func)
 {
 	import std.format : format;
 	enum fallback = q{
-		import std.range : back, dropBackOne;
-		import dub.internal.vibecompat.core.log : logDebug;
-		scope (failure)
+		import dub.internal.vibecompat.core.log : logDiagnostic;
+
+		Exception firstEx;
+		try
+			return m_suppliers[0].%1$s(args);
```

Comment:
Won't this trigger and RangeError if all suppliers failed?

Reply:
Interesting point! When all suppliers failed the exception will be rethrown and quit the program.